api/axis.py

```diff
@@ -1377,14 +1401,22 @@ def process_one(self, e, ns):
                     **kwargs
                 )
             elif transaction_id:
-                # This is an audiobook. If necessary we
-                # are prepared to make another API request to
-                # turn the transaction ID into a Findaway
-                # audio manifest.
-                fulfillment = AudiobookFulfillmentInfo(
+                # If this is an audiobook, we'll need to make two more
+                # requests: the first to get the Findaway content ID,
+                # license ID, and session key; the second to get the
+                # audiobook metadata.
+                #
+                # If this is an ebook being delivered in AxisNow
+                # format, we'll need to make one more request, to get
+                # the Book Vault UUID and ISBN.
```

Comment:
Does this request happen after the first two requests or are two above just for Findaway?

Reply:
Oh nevermind. It wasn't clear at first that it was audiobook vs ebook.

Reply:
Audiobook means two more requests, AxisNow means one more request. I'll clarify.